encord/user_client.py

```diff
@@ -306,7 +307,7 @@ def create_project(
             project_description: the optional description of the project
             ontology_hash: the uid of an ontology to be used. If omitted, a new empty ontology will be created
             workflow_settings: selects and configures the type of the quality control workflow to use, See :class:`encord.orm.project.ProjectWorkflowSettings` for details. If omitted, :class:`~encord.orm.project.ManualReviewWorkflowSettings` is used.
-
+            workflow_template_hash: project will be created using a workflow created from the template provided
```

Comment:
Can you double check with @david-babuschkin-encord whether this is the way we're documenting the switch from TMS1 to workflow projects?

Reply:
```suggestion
            workflow_template_hash: project will be created using a workflow based on the template provided.
```

Reply:
@denis-cord  We link clients to our workflow docs in other places, so that when they arrive at the point where they are using this method they should be aware of what workflows are. Justin and I agreed to avoid talking about a 'transition' from TMS 1.0 -> 2.0 and just communicate it as a new, more powerful option.